app/models/assignment_repo/creator/reporter.rb

```diff
@@ -34,10 +34,14 @@ def broadcast_message(type: :text, message:, status_text:, repo_url: nil)
 
       # Reports the elapsed time to Datadog
       #
-      def report_time(start_time)
+      def report_time(start_time, assignment)
         duration_in_millseconds = (Time.zone.now - start_time) * 1_000
-        GitHubClassroom.statsd.timing("v2_exercise_repo.create.time", duration_in_millseconds)
-        GitHubClassroom.statsd.timing("exercise_repo.create.time", duration_in_millseconds)
+        if group_assignment.starter_code?
+          GitHubClassroom.statsd.timing("exercise_repo.create.time.with_importer", duration_in_millseconds)
```

Comment:
cc @shaunakpp how does this look? Using v1 for the new stat and keeping v1/v2 for the existing ones?

Reply:
:+1: To this

Reply:
yes that looks good 👍